launch/launch/launch_service.py

```diff
@@ -413,3 +413,7 @@ def shutdown(self, force_sync=False) -> Optional[Coroutine]:
     def context(self):
         """Getter for context."""
         return self.__context
+
+    @property
+    def event_loop(self):
+        return self.__loop_from_run_thread
```

Comment:
nitpick: Not sure if it's needed (like if a linter will complain) but there's no docstring here.

Reply:
fixed https://github.com/ros2/launch/pull/528/commits/38098a6c43261ec60be9ed6a38930c7d57144d26